docs/Inputs.md

```diff
@@ -168,22 +168,14 @@ This input does not handle `null` values. You would need the `<NullableBooleanIn
 
 You can use the `options` prop to pass any option supported by the Material UI `Toggle` components.
 
-{% raw %}
```

Comment:
Why did you remove this one? If it's not good, you should replace it with another one.

Reply:
Because the `labelPosition` prop does not exist anymore and we have yet to decide how to handle it, if possible

Reply:
Oh and by default, it is display at the right of the Switch